src/main/java/mate/controller/manufacturer/DeleteManufacturerController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller.manufacturer;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/delete")
+public class DeleteManufacturerController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) INJECTOR.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long manufacturerID = Long.valueOf(req.getParameter("id"));
+        manufacturerService.delete(manufacturerID);
```

Comment:
After the deletion of the manufacturer, there is no redirection or response being sent to the client. You should redirect to a confirmation or another relevant page.